projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-rectangular.php

```diff
@@ -268,6 +289,54 @@ public function calculate_image_sizes( $group ) {
  * Jetpack tiled row class.
  */
 class Jetpack_Tiled_Gallery_Row {
+	/**
+	 * Groups of images
+	 *
+	 * @var array
```

Comment:
The constructor's parameter declaration says `object`, but  probably should actually say `object[]`.

Reply:
Done in b503fca7d2b44b8c41de15263072a4145cc1b69c